test/util/util.c

```diff
@@ -497,7 +497,7 @@ connect_stream_with_tls (mongoc_ssl_opt_t *ssl_opt,
                          int32_t connecttimeoutms,
                          bson_error_t *error)
 {
-   mongoc_stream_t *stream;
+   mongoc_stream_t *stream = NULL;
```

Comment:
Resolved a clang-tidy warning locally. If `goto done` had run before `stream` was initialized, `mongoc_stream_destroy (stream);` would be given an uninitialized `stream`.